0-bootstrap/README.md

```diff
@@ -103,9 +103,30 @@ To enable automatic creation of the [required groups](https://cloud.google.com/a
 - Grant role `roles/serviceusage.serviceUsageConsumer` to the user running Terraform on the billing project.
 - Provide values for the groups and billing project in the variable `groups`.
 
-All groups in the `groups.required_groups` are required.
+All groups in the `groups.required_groups` are **required**.
+
+All groups in the `groups.optional_groups` are **optional** and the roles assigned are:
```

Comment:
Since we are listing all the roles for optional groups, why don't we do the same for required ones as well?

Reply:
On the code we don't assign any role for the required groups.